build.assets/Makefile

```diff
@@ -554,3 +554,48 @@ build-centos7-assets:
 		--build-arg DEVTOOLSET=$(DEVTOOLSET) \
 		--tag $(BUILDBOX_CENTOS7_ASSETS)-$(RUNTIME_ARCH) \
 		-f Dockerfile-centos7-assets .
+
+#################################################################################
+# Multiarch build steps
+#################################################################################
+
+GCC_VERSION ?= 10.4.0
+CLANG_10_VERSION ?= 10.0.1
+CLANG_7_VERSION ?= 7.0.1
```

Comment:
nit: I don't think we need this as a Makefiles variable. I don't think we will update it any time soon.